### PR TITLE
Added Azure auth and minor typo/link fixes

### PR DESCRIPTION
**Pull request by rem20806**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/787)

### CODE REVIEW

**Comment by MaanavD** [comment]
I am ok with my commits being contributed.

**Comment by mohona17** [comment]
I am also okay with my commits being contributed.

File: content/docs/azure/authentication.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
+description = "Restrict access of your deployment to specified IP addresses"
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: Recommendation: Add the word "Azure" a bit earlier on the page, so people know this doc applies specifically to Kubeflow on Azure - either the description in the front matter, or the first sentence on the page, are good places. (Even though the technique described here can probably be used across other cloud providers too, it's good to remind readers that this is the way currently recommended for Azure but not necessarily for other clouds.)

Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: Recommendation: It's a good idea to give people a hint about where a link will take them, especially if the link goes outside the current doc set. Change `[this guide]` to something like `[the Kubernetes guide to configuring a firewall]` or whatever words best describe the doc publisher and topic.

File: content/docs/azure/authentication.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer by using `spec.loadBalancerSourceRanges`. This is currently supported on all major cloud providers.
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: s/Service/service

(For consistent capitalization on the page.)

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-787--competent-brattain-de2d6d.netlify.com/docs/azure/authentication/

**Comment by sarahmaddox** [comment]
Thanks for this contribution @rem20806!  A couple of hints to make things easier for reviews:

* Add the link(s) to the Netlify preview, pointing directly to the page(s) that you've modified.
* Update the PR name to describe its scope better.

File: content/docs/azure/authentication.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  ports:
+  - port: 8765
+    targetPort: 9376
+  selector:
+    app: example
+  type: LoadBalancer
+  loadBalancerSourceRanges:
+  - 10.0.0.0/8
+```
+#### External IP Addresses
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: s/Addresses/addresses

(For consistent capitalization on the page)

File: content/docs/azure/authentication.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
+
+
+When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer by using `spec.loadBalancerSourceRanges`. This is currently supported on all major cloud providers. 
+
+### Editing the LoadBalancer service
+Use the `kubectl edit svc <loadbalancer-name> -n kubeflow` to add your source ranges. Pressing `i` will allow you to insert text, and `esc` followed by `:wq` will allow you to write and quit out of the text editor.
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: This looks like the text editor is VI or something similar. Does the `kubectl edit` utility always invoke the same editor, no matter which OS the person is using, and no matter what they've set as their default editor? (I haven't tried it, so I'm asking from a user's point of view. I've set my default editor on my Mac and Linux machines to Pico. I don't use Windows for Kubeflow.)

File: content/docs/azure/authentication.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+description = "Restrict access of your deployment to specified IP addresses"
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
+
+
+When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer by using `spec.loadBalancerSourceRanges`. This is currently supported on all major cloud providers. 
+
+### Editing the LoadBalancer service
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: How about using a heading level 2 for all headings on this page?

File: content/docs/azure/authentication.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
```
Comment (sarahmaddox) [inline content/docs/azure/authentication.md]: Recommendation: Reword the title, as it currently implies that we're restricting auth as well as restricting access.

**Comment by rem20806** [comment]
@sarahmaddox I made the requested changes and added a link describing the editor options for kubectl edit function. Please let me know if you see any other issues. Also, where do I find the link for the preview for next time?

**Comment by rem20806** [comment]
https://deploy-preview-787--competent-brattain-de2d6d.netlify.com/

**Comment by sarahmaddox** [comment]
Thanks for the updates and for the CLA confirmations!

/lgtm
/approve

**Pull request merged**